### ISSUE TITLE
No items when use appium-xcuitest-driver 2.5+

### ISSUE THREAD

**Issue by QAutomatron** [mykola-mokhnach/Appium-iOS-Inspector#16]
By default Appium 1.6.2 use appium-xcuitest-driver 2.4.2 and inspector works great.

But if update appium-xcuitest-driver to newest version 2.5.1 or 2.5.0. Inspector show no items in tree.

<img width="1038" alt="screen shot 2017-01-04 at 17 27 13" src="https://cloud.githubusercontent.com/assets/15247795/21645460/18c2b11a-d2a3-11e6-81f4-d36f0a6a8d65.png">

**Comment by mykola-mokhnach** [comment]
What is in browser console?

**Comment by QAutomatron** [comment]
Console: 

```
Logger.js:36 INFO :Inspector.initSessionId:14 initSessionId...
jquery-1.js:8492 Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
send @ jquery-1.js:8492
Logger.js:36 INFO :Object.<anonymous>:26 success
Logger.js:36 INFO :Object.<anonymous>:26 [object Object]
Logger.js:36 INFO :Object.<anonymous>:26 Session Id: f56328fd-5a75-44fb-9d84-129895f1ca5c
Logger.js:36 INFO :Inspector.initAutXml:14 initAutXml...
Logger.js:36 INFO :Object.<anonymous>:26 success
Logger.js:36 INFO :Object.<anonymous>:26 [object Object]
Logger.js:36 INFO :Inspector.initScreenshot:14 initScreenshot...
iOS Inspector.html:24 before calling resize()
Logger.js:36 INFO :Object.<anonymous>:26 Screenshot received. Scheduling structure update...
inspector1.js:39 resized to 428
inspector1.js:39 resized to 428
```

**Comment by QAutomatron** [comment]
After update Inspector to last commit ("Fix for #15") everything works fine. Sorry for bothering you.

**Comment by mykola-mokhnach** [comment]
np. I'm always happy to improve the tool.
